types/jquery/README.md

```diff
@@ -0,0 +1,53 @@
+### Usage
+
+#### Global
+
+When jQuery is globally available, you can use `jQuery` and `$` directly.
+
+#### Importing (with a global DOM available)
+
+When you want to import jQuery as a module and have a global DOM available (e.g. browser and browser-like environments):
+
+```typescript
+import * as jQuery from 'jquery';
```

Comment:
Hm... `import jQuery = require('jquery')` is probably a better example.

`import * as jQuery from 'jquery'` is most certainly invalid under the currently proposed commonjs interop spec; but TypeScript doesn't have a better emit for the would-be correct `import jQuery from 'jquery'` yet.